### PR TITLE
Update Runs Page

### PR DESCRIPTION
**Pull request by laurapacilio**
### What
We updated the Runs page in workspaces to allow users to filter the runs list by status, operation, and source - hooray! We [updated the API docs](https://github.com/hashicorp/terraform-docs-common/pull/44), but we also need to update this page as well :-) This PR updates the outdated **Navigating Runs** section of this page.

It also removes excess screenshots that are out-of-date and fixes the rest of the page copy so that it better aligns with our style guide. :-) 

### Why
Fix the outdated docs to align with the current product!

----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [x] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
- [x] Description links to related pull requests or issues, if any.

#### Content
- [x] Redirects have been added for moved, renamed, or deleted pages. This requires a separate PR in the [`terraform-website` repository](https://github.com/hashicorp/terraform-website) `redirects.next.js` file.
- [x] API documentation and the API Changelog have been updated. 
- [x] Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [x] Pages with related content are updated and link to this content when appropriate.
- [x] Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [x] New pages have metadata (page name and description) at the top.
- [x] New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [x] New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [x] UI elements (button names, page names, etc.) are bolded.
- [ ] The Vercel website preview successfully deployed.

#### Reviews
- [ ] I or someone else reviewed the content for technical accuracy.
- [ ] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

File: website/docs/cloud-docs/run/manage.mdx [hunk 1]
```diff
@@ -7,36 +7,25 @@ description: >-
...
 
-Each workspace has two ways to view and navigate runs:
-
-- A "Runs" link, which goes to the full list of runs.
-- A "Current Run" link, which goes to the most recent active run. (This might not be the most recently initiated run, since runs in the "pending" state remain inactive until the current run is completed.)
-
-![runs list](/img/docs/runs-list.png)
-
-From the list of runs, you can click to view or interact with an individual run.
+Go to the **Runs** tab in the workspace to review a list of all Terrform runs associated with the workspace.
```
Comment (laurapacilio) [inline website/docs/cloud-docs/run/manage.mdx:18]:
```suggestion
Click the **Runs** tab to review a list of all Terrform runs associated with the workspace.
```

**Review by mjyocca — APPROVED** [review]
LGTM 👍

**Review by mgarrell777 — APPROVED** [review]
Hey @laurapacilio I'm Matthew, new technical writer on Terraform ;)

Thank you for your efforts here. Just a few minor suggestions. But otherwise looks good.

File: website/docs/cloud-docs/run/manage.mdx [hunk 2]
```diff
@@ -1,83 +1,74 @@
...
-An individual run page shows the progress and outcomes of each stage of the run.
-
-![a run page](/img/docs/runs-run-page.png)
-
-Most importantly, it shows:
+Click a run to go to its details page. The details page contains the following information:
 
 - The current status of the run.
 - The code commit associated with the run.
 - How the run was initiated, when, and which user initiated it (if applicable).
```
[the remark this answers is not in the document]
Comment (laurapacilio) [reply website/docs/cloud-docs/run/manage.mdx:24]: YES - thank you!

File: website/docs/cloud-docs/run/manage.mdx [hunk 3]
```diff
@@ -1,83 +1,74 @@
...
 
-Canceling a run is roughly equivalent to hitting ctrl+c during a Terraform plan or apply on the CLI. The running Terraform process is sent an INT signal, which instructs Terraform to end its work and wrap up in the safest way possible. (This gives Terraform a chance to update state for any resources that have already been changed, among other things.)
+Canceling a run is roughly equivalent to typing ctrl+c during a Terraform plan or apply on the CLI. The running Terraform process is sent an INT signal, which instructs Terraform to end its work, update state for any resources that have already been changed, and wrap up in the safest way possible.
 
-In rare cases, a cancelled run can fail to end gracefully, and will continue to lock the workspace without accomplishing anything useful. These stuck runs can be **force-canceled,** which immediately terminates the running Terraform process and unlocks the workspace.
+In rare cases, a cancelled run can fail to end, continuing to lock the workspace. You can forcefully cancel these runs, which immediately terminates the running Terraform process and unlocks the workspace.
```
[the remark this answers is not in the document]
Comment (laurapacilio) [reply website/docs/cloud-docs/run/manage.mdx:56]: YESSSSSSSS thank you for catching this!

File: website/docs/cloud-docs/run/manage.mdx [hunk 4]
```diff
@@ -1,83 +1,74 @@
...
+If a run is currently planning or applying, users with [permission to apply runs](/cloud-docs/users-teams-organizations/permissions#general-workspace-permissions) for the workspace can click **Cancel Run** to stop the run before it finishes.
 
 [permissions-citation]: #intentionally-unused---keep-for-maintainers
 
-Canceling a run is roughly equivalent to hitting ctrl+c during a Terraform plan or apply on the CLI. The running Terraform process is sent an INT signal, which instructs Terraform to end its work and wrap up in the safest way possible. (This gives Terraform a chance to update state for any resources that have already been changed, among other things.)
+Canceling a run is roughly equivalent to typing ctrl+c during a Terraform plan or apply on the CLI. The running Terraform process is sent an INT signal, which instructs Terraform to end its work, update state for any resources that have already been changed, and wrap up in the safest way possible.
```
[the remark this answers is not in the document]
Comment (laurapacilio) [reply website/docs/cloud-docs/run/manage.mdx:54]: yes! love this - good call out!

**Pull request merged**